monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md

```diff
@@ -0,0 +1,78 @@
+
+Sin embargo, al ser completamente nuevo, Bitcoin tenía fallas técnicas, incluyendo mala privacidad. Y así, otra persona o grupo seudónimo llamado Nicolas van Saberhagen desarrollaron el protocolo CryptoNote. Los programadores se dieron cuenta y nació Monero. Monero resolvió muchos de los inconvenientes de Bitcoin, incluyendo su fácil vigilancia por parte de gobiernos y empresas. Tal como declaro la compañía de análisis de blockchain Chainalysis recientemente en un seminario web público, [chainalysis.com/.../Advanced-Obfuscation-Techniques-Public-Webinar.mp4font](https://go.chainalysis.com/rs/503-FAP-074/images/Advanced-Obfuscation-Techniques-Public-Webinar.mp4), “Monero está mucho más enfocado en la privacidad que sus semejantes. Todo en Monero está ofuscado, no se puede distinguir nada.  
+
+Monero es completamente de código abierto y tiene una de las comunidades más activas y extendidas que cualquier otra criptomoneda. La comunidad se ha ganado la reputación de tener unos valores que apoyan la autodeterminación, la soberanía individual y la protección de los derechos humanos. Si estás buscando un sistema que pueda tomar el poder de gobiernos y banqueros centrales corruptos, y que a la larga mejore la forma en que las personas interactúan y trancen entre sí, te invitamos a usar Monero.  
+
+Cuando se hace una transacción usando Monero, el que envía, el que recibe y la cantidad enviada son ocultas utilizando criptografía, evitando que los gobiernos, empresas y otras personas vean y graben tus acciones. Con Monero, la compra y venta individual recibe la protección que tu esperas obtener al hacer actividadesprivadas en tu propia casa. Las matemáticas y el software de código abierto te protegen, no los bancos, tampoco el sistema legal, ni la policía.  
+
+Sea que necesites mover tu dinero a alguna reserva de valor privada por necesidad o quieras por principios optar salirte del actual sistema financiero, únete a nosotros. Juntos podemos usar nueva tecnología para limitar la intromisión de gobiernos y bancos corruptos, y participar en un sistema financiero que empodere a las personas. Estamos todos en esto. Usemos estas herramientas para ser el cambio que queremos ver para nosotros, las futuras generaciones y el mundo.  
+
+No esperamos que tomes nuestra palabra para nada de esto y te motivamos ha que hagas tus propias investigaciones y revises con fuentes. Compartiremos algunas referencias que recomendamos por donde comenzar. Una vez hayas terminado de hacer tu propia investigación, vuelve para ver la guía rápida de como comenzar a usar Monero (ver más abajo).  
```

Comment:
"Te motivamos Ha que hagas" 

Ha van sin h? por preterito y el verbo hacer?

Reply:
Yes you're right. "a" is not with an "h". Removed. Fixed.